transformer_engine/jax/__init__.py

```diff
@@ -23,7 +23,6 @@
     'NVTE_FP8_COLLECTION_NAME',
     'fp8_autocast',
     'update_collections',
-    'update_fp8_metas',
```

Comment:
Also tell in the description that this function is now removed and not needed as fully automatic?
This is part of the API change this PR is doing.

Reply:
This function has not used for a while, after we migrated to `custom_partitioning`. Now FP8 meta is updated automatically in the backward, therefore no needed this function. Also added to `Changes`.